pkg/autoscaler/fake/fake_metric_client.go

```diff
@@ -58,3 +77,24 @@ var StaticMetricClient = MetricClient{
 	StableRPS:         10.0,
 	PanicRPS:          10.0,
 }
+
+// Endpoints is used to create eps
```

Comment:
nit: comments should be sentences and end in periods. I would also avoid abbreviations -- what is eps?